worker/lib/helpers/graphics.ts

```diff
@@ -113,15 +113,7 @@ export default class ScreenCapture {
 				};
 				const exitHandler = () => {
 					clean();
-
-					resolve(
-						pack(this.destination, {
-							map: header => {
-								header.name = basename(header.name);
-								return header;
-							},
-						}).pipe(createGzip()),
-					);
```

Comment:
Don't understand the reason for this. Is it because we are using a volume now and no need to gzip the photos captured.

Reply:
thats right @vipulgupta2048